core/subscription/service_test.go

```diff
@@ -646,6 +649,259 @@ func TestService_Search(t *testing.T) {
 		assert.Equal(t, "1", pbtypes.GetString(resp.Records[0], bundle.RelationKeyName.String()))
 		assert.Equal(t, "1", pbtypes.GetString(resp.Records[0], bundle.RelationKeyId.String()))
 	})
+
+	t.Run("SubscribeGroup: tag group", func(t *testing.T) {
+		fx := newFixture(t)
+
+		source := "source"
+		spaceID := "spaceId"
+		relationKey := "key"
+		subID := "subId"
+		collectionID := "collectionId"
+
+		fx.collectionService.EXPECT().SubscribeForCollection(collectionID, subID).Return([]string{"1"}, nil, nil).Times(1)
+		fx.collectionService.EXPECT().UnsubscribeFromCollection(collectionID, subID).Return().Times(1)
+
+		defer fx.a.Close(context.Background())
+		defer fx.ctrl.Finish()
+		key, err := domain.NewUniqueKey(smartblock.SmartBlockTypeObjectType, source)
+		assert.Nil(t, err)
+
+		fx.store.EXPECT().GetDetails(source).Return(&model.ObjectDetails{
```

Comment:
it's better to use real object store (newFixtureWithRealObjectStore) and real kanban service

Reply:
Fixed